packages/commons/src/logging/loggerMiddleware.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import * as express from "express";
+import { LoggerMetadata, logger } from "./index.js";
+
+export function loggerMiddleware(serviceName: string) {
+  return (req: any, res: any, next: express.NextFunction): void => {
+    const loggerMetadata: LoggerMetadata = {
+      serviceName,
+      userId: req.ctx?.authData?.userId,
+      organizationId: req.ctx?.authData?.organizationId,
+      correlationId: req.ctx?.correlationId,
+    };
```

Comment:
Could we remove the any?
My proposal:

```suggestion
export function loggerMiddleware(serviceName: string): express.RequestHandler {
  return (req, res, next): void => {
    const context = (req as express.Request & { ctx?: AppContext }).ctx;

    const loggerMetadata: LoggerMetadata = {
      serviceName,
      userId: context?.authData?.userId,
      organizationId: context?.authData?.organizationId,
      correlationId: context?.correlationId,
    };
```

This way, if the `AppContext` changes we would be "notified" by typescript at compile time